features/navigator-d-d-g.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file name makes me want to cry.

@SlayterDev I think we should change this to `navigatorInterface` (interface is the lingo specifications use to describe these kinda things)

Reply:
@jonathanKingston I'll update the PR, I think `navigatorInterface` is better anyway, regardless of weird filenames